src/web/js/dashboard/GoogleAPI.js

```diff
@@ -14,48 +14,58 @@ class GoogleAPI {
   /**
    *  Sign in the user upon button click.
    */
-  signIn = (event) => {
-    return gwrap.load({name: 'drive',
+  signIn = () => {
+    return window.gwrap.load({name: 'drive',
       version: 'v3',
       reauth: {
         immediate: false
       }
     });
   }
 
-  /**
-   *  Sign out the user upon button click.
-   */
-  signOut = (event) => {
-    throw "Can't sign out yet.";
-  }
+  removeAllAppDataFiles = () => {
+     return window.gapi.client.drive.files.list({
+       spaces: 'appDataFolder'
+     }).then(resp => {
+       resp.result.files.forEach(file => {
+         window.gapi.client.drive.files.delete({'fileId': file.id}).then(console.log);
+       });
+     });
+   }
 
-  createAppFolder = (appName) => {
```

Comment:
we decided to remove the parameter because in this project the only thing this will ever be used with is the "pyret" parameter, so we figured it would be a bit cleaner to write it this way.